recipes/pyldavis/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyldavis" %}
```

Comment:
I think you need to change this to `pyLDAvis` so that the download link is correct:

see:
https://pypi.org/project/pyLDAvis/#files